db/migrate/20181206153510_add_home_page_to_legislation_processes.rb

```diff
@@ -0,0 +1,16 @@
+class AddHomePageToLegislationProcesses < ActiveRecord::Migration
+  def change
+    add_column :legislation_processes, :homepage, :text
```

Comment:
Adding the translation column is enough :wink:, so in my humble opinion this line (and the one in the `schema.rb` file) should be removed. The reason we still keep both original and translation columns in other tables is we've only added translations for other models recently, but we're planning to remove the original columns in a future release.

Reply:
ok